### PR TITLE
Ems 114494867 - Added expectation handling for nil MP3 recordings. 

### PR DESCRIPTION
**Pull request by eschlange**
* Updated responses controller to handle nil MP3 recordings.
* Added specs for new logic.

[#114494867]

### CODE REVIEW

File: app/controllers/responses_controller.rb [hunk 1]
```diff
@@ -2,21 +2,16 @@
 class ResponsesController < ApplicationController
   protect_from_forgery except: :create
 
+  RECORDING_ERROR_MESSAGE = "ERROR: Ensure microphone is enabled.".freeze
```
Comment (michaelwehrley) [inline app/controllers/responses_controller.rb]: Note for the future: @ericcf found that if you place this comment `# frozen_string_literal: true` at the top of your file, you don't need to freeze string literals, they are already 'frozen'

**Comment by michaelwehrley** [comment]
Great spec coverage!  Thanks!

File: app/controllers/responses_controller.rb (hunk 1 above)
Comment (eschlange) [inline app/controllers/responses_controller.rb]: Cool, thanks!

Comment (ericcf) [inline app/controllers/responses_controller.rb]: @michaelwehrley @eschlange that's only true if you're using Ruby 2.3.0.

**Pull request merged**